### DISCUSSION TITLE
Navigation menu issue

### DISCUSSION THREAD

**Discussion by ahmedabbous** [TDRA-ae/aegov-dls#33] [General]
Hello,
Kindly note there is issue with the navigation menu on mobile (only Iphone devices) where the arrows is not showing, with this issue there is no way to access the sub links 

Thank you

**Comment by moiz-conceptlz** [comment]
Hi @ahmedabbous 

Please follow https://github.com/TDRA-ae/aegov-dls/issues/35 for further details and updates.
